src/incubator/Dialog/__tests__/index.new.spec.tsx

```diff
@@ -84,6 +84,7 @@ describe('Incubator.Dialog sanity checks', () => {
     });
     expect(dialogDriver.getModal().isVisible()).toBeFalsy();
   });
+
   it('Should exist only if visible', async () => {
```

Comment:
I don't think async is needed here. I probably missed it because I just copied the old tests and change it to the driver.
I also see that this test that I added is not good because I misunderstood the behavior. I'll create a pr to fix.
```suggestion
  it('Should exist only if visible', () => {
```